helpers/class.File.php

```diff
@@ -112,6 +112,28 @@ public static function isFileInsideDirectory($filename, $directory)
         if (false === $canonicalFilename) {
             return false;
         }
+
+        return 0 === strpos($canonicalFilename, $canonicalDirectory);
+    }
+
+    /**
+     * Helps prevent 'path traversal' attacks
+     *
+     * @param string $filename  absolute path to file
+     * @param string $directory absolute path to directory
+     * @return bool
+     */
+    public static function isAbsoluteFileInsideDirectory($filename, $directory)
```

Comment:
I'd ask to add some test cases for the method, but I realize php-vfs won't be able to help you with `realpath`, and the helper is in the global namespace, so you won't even be able to mock the entire function :/

Reply:
I totally agree. I'll figure out what to do. Probably integration test then.